debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -2952,6 +2952,7 @@ functionNameBase
     | JSON_VALID | JSON_TABLE | JSON_SCHEMA_VALID | JSON_SCHEMA_VALIDATION_REPORT
     | JSON_PRETTY | JSON_STORAGE_FREE | JSON_STORAGE_SIZE | JSON_ARRAYAGG
     | JSON_OBJECTAGG
+    | STATEMENT
```

Comment:
I also don't believe this is the right place for this change, it should be part of `keywordsCanBeId` rule.

Furthermore, this `keywordsCanBeId` already includes the `STATEMENT` keyword as it was added sometime ago specifically for MariaDB, so I don't think this change is needed based on the current `main` code base.

I would again redirect you to my above inquiry, does the DDL tests pass by simply adding the SQL but no change to the grammar.